aws/compute_engine/instance_template/ssh/key.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDH/L16ACajBGASbwvPw/9kgCchE2WsPCmXR44cG4yqojPj3gSNOkxCwAcbbSraru2UxPSZzlb08WFRYz2Ewrxs1hD7A7v9CSA2P1Kxc5mRkuH/p0svIwaq/pUOvNWdEREnvVELScRhjUqtPYil3XOXQBrNNR8EHeiJ4bt04oQB3sr71HFtm65hbB3OnGm5Zymgq3vREKCaac2VoUQJ+Iw4E81sMrxS9Y6kAL+iNVZfcJYmANc+QJyyAMIZRTtjoTKiDeQIpSPX5M7lG7MV26bXIoysY1ISTu9ZB6kihYyiAXGAN1asLU16+9g1HzkHUJAVtEvOeBOQt2lAaHD6RAjkjrCyDCmEgczMT6x+P5ZiX6ffN4B1BTQArOxVXPGWP73hmxkiztRbtyKQincy9BlGyNxvUSnnrk2X9VjnQDRlgww7Qe7aEpjB0PHcnjQW6/FTRu7NR8K6t+VDV4XnsWSNekraI6hAIuXJbdiQaTc3H1CsRRLVNE2Spq7E5vvbGds= dmitrogavrichuk@iMac-Dmitro.local
```

Comment:
Following from other comment, this file is not needed in the repo if the key location is a variable